recipes/rmlui/3.3/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class RmluiConan(ConanFile):
+    name = "rmlui"
+    description = "RmlUi - The HTML/CSS User Interface Library Evolved"
+    homepage = "https://github.com/mikke89/RmlUi"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "MIT"
+    topics = ("conan", "css", "gui", "html", "lua", "rmlui")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "build_lua_bindings": [True, False],
```

Comment:
I would name this option `with_lua`. A user of `rmlui` wants to use this package with or without lua.

Reply:
I do understand the sentiment to stick with the "with_" prefix. With that being said, I'd rather go with "with_lua_bindings" because of the added meaning that it comes with bindings and the option isn't just about including Lua or not. Would you be okay with this?

Reply:
I'm fine with this. I'm not too feeling too sentimental about this 😄